docs/source/developer/setup.rst

```diff
@@ -150,6 +150,96 @@ be able to use it with openwisp-radius (and openwisp-radius) easily:
     ./manage.py migrate --fake openwisp-radius 0001_initial_freeradius
     ./manage.py migrate
 
+Automated periodic tasks
+------------------------
+
+Some periodic commands are necessary to enable certain
+features and facilitate database cleanup.
+There are two ways to automate these tasks:
+
+1. Celery-beat (Recommended Method)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, you need to create a `celery configuration file as it's created in example <https://github.com/openwisp/openwisp-radius/tree/master/tests/openwisp2/celery.py>`_
+
+Add celery to ``__init__.py`` of your project:
+
+.. code-block:: python
+
+    from .celery import app as celery_app
+
+    __all__ = ['celery_app']
+
+In the settings.py, configure the ``CELERY_BEAT_SCHEDULE`` as `done in the example <https://github.com/openwisp/openwisp-radius/tree/master/tests/openwisp2/settings.py#L141>`_.
+
+.. note::
+    Some celery tasks take an argument, example ``365`` is given here for
+    ``delete_old_radacct``. These arguments are passed to their respective management
+    commands. More information about these parameters can be
+    found at the `management commands page </user/management_commands.html>`_.
+
+Next, `install docker-compose <https://docs.docker.com/compose/install/>`_
+and start redis-server inside docker:
```

Comment:
it looks like running docker compose is required to use this feature, I think we should make it explicit that this is a suggestion, only a redis instance is required, the user may already have a cluster and will surely wonder if they can reuse the cluster they already have.

Reply:
Fixed this. 
And removed the docker-compose file and moved to docker only, since docker-compose maybe overkill for using just one container.